src/FACT_internal.c

```diff
@@ -628,8 +630,6 @@ uint8_t FACT_INTERNAL_CreateSound(FACTCue *cue, uint16_t fadeInMS)
 
 				newSound->tracks[i].events[j].timestamp =
 					newSound->sound->tracks[i].events[j].timestamp;
-				newSound->tracks[i].events[j].loopCount =
```

Comment:
We'll need to set loopCount to 0 by default, for event types that don't have a loop/repeat value, otherwise we'll get a branch on uninitialized value.